dist/main.js

```diff
@@ -0,0 +1,20 @@
+const render = new Renderer()
+const expenseManager = new ExpenseManager()
+expenseManager.getAllExpenses().then(dbData => render.renderExpenses(dbData))
```

Comment:
A better practice would be putting this line in a `loadData` function and invoke that function. Also maybe doing that inside that function with `async` and `await`